pkg/ec2metadatawrapper/ec2metadatawrapper_test.go

```diff
@@ -0,0 +1,78 @@
+/*
+  Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License").
+  You may not use this file except in compliance with the License.
+  A copy of the License is located at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  or in the "license" file accompanying this file. This file is distributed
+  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+  express or implied. See the License for the specific language governing
+  permissions and limitations under the License.
+*/
+
+package ec2metadatawrapper
+
+import (
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws/ec2metadata"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	// iidRegion is the instance identity document region
+	iidRegion = "us-east-1"
+)
+
+var testInstanceIdentityDoc = ec2metadata.EC2InstanceIdentityDocument{
+	PrivateIP:        "172.1.1.1",
+	AvailabilityZone: "us-east-1a",
+	Version:          "2010-08-31",
+	Region:           "us-east-1",
+	AccountID:        "012345678901",
+	InstanceID:       "i-01234567",
+	BillingProducts:  []string{"bp-01234567"},
+	ImageID:          "ami-12345678",
+	InstanceType:     "t2.micro",
+	PendingTime:      time.Now(),
+	Architecture:     "x86_64",
+}
```

Comment:
this is fine I suppose but not strictly necessary since the identity doc stuff isn't used

Reply:
removed